torch/_dynamo/variables/torch.py

```diff
@@ -374,21 +374,26 @@ def get_state_from_generator():
                 **options,
             )
         if (
-            any([isinstance(x, DynamicShapeVariable) for x in args])
-            and self.value == math.sqrt
+            all([isinstance(x, DynamicShapeVariable) for x in args])
         ):
-            from torch.fx.experimental.symbolic_shapes import sym_sqrt
+            if self.value == math.sqrt:
+                from torch.fx.experimental.symbolic_shapes import sym_sqrt
+                fn_ = sym_sqrt
+            else:
+                fn_ = self.value
 
-            return TensorVariable.create(
+            out = TensorVariable.create(
                 tx=tx,
                 proxy=tx.output.create_proxy(
                     "call_function",
-                    sym_sqrt,
+                    fn_,
                     *proxy_args_kwargs(args, kwargs),
                     current_tx=tx,
                 ),
                 **options,
             )
+            assert isinstance(out, DynamicShapeVariable)
```

Comment:
It looks like `TensorVariable.create()` can return a bunch of different types depending on the op that we're running.

Reply:
hmm removing this assert because the func could also be tensor constructor in which case the out type is TensorVariable